packages/web-components/fast-foundation/src/accordion/accordion.ts

```diff
@@ -108,6 +122,7 @@ export class Accordion extends FASTElement {
         if (this.isSingleExpandMode()) {
             this.resetItems();
             event.target.expanded = true;
+            event.target.setAttribute("aria-disabled", "true");
```

Comment:
I'm not sure this is correct - I think we only want this disabled if it should be disabled. The control should be intractable by default. Disabled as an attribute would set aria-disabled and prevent user interaction.